src/rpc.rs

```diff
@@ -44,7 +44,7 @@ impl RpcClient for UPClientZenoh {
 
         // Create UAttributes and put into Zenoh user attachment
         let uattributes = UAttributes::request(
-            UUIDBuilder::new().build(),
+            UUIDBuilder::build(),
             topic,
             self.get_response_uuri(),
             options.clone(),
```

Comment:
Or perhaps... slightly different question, if we call `invoke_method()` once, are we able to call `invoke_method()` once again immediately thereafter, or would this line of code:
```rust
        let getbuilder = self
            .session
```
block the second invocation until the first had returned?

Sorry, again, I'm a newb to Zenoh :)

Reply:
If we want to call `invoke_method` twice at the same time, I think we can use something like `async_std::task::spawn`. Then the two `invoke_method` will not interfere with each other.

If we have something like `invoke_method(...).await`, the second one still needs to wait for the first one.

Reply:
Thx. Makes sense